docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -57,19 +48,79 @@ $ uname -r
 </tbody>
 </table>
 
-## Step 1/3. Install and configure Teleport node
+(!docs/pages/includes/tctl.mdx!)
+
+## Background
+
+Teleport Nodes submit SSH session traffic to the Auth Service for
+storage, copying what is echoed in a terminal. These recorded sessions can be
+replayed later via the `tsh play` command or in the Teleport Web UI.
+
+The advantage of this approach is that, since no input is captured, these
+session recordings do not contain passwords that were entered into a terminal.
+
+The disadvantage is that session recordings can be bypassed using several
+techniques:
 
-Follow our [installation instructions](../../installation.mdx) to install the Teleport Auth Service, Proxy Service, and Nodes. To get started with Teleport Cloud, sign up for a [free trial](https://goteleport.com/signup/). 
+- **Obfuscation**. For example, even though the command
+  `echo Y3VybCBodHRwOi8vd3d3LmV4YW1wbGUuY29tCg== | base64 --decode | sh` does not contain
+  `curl http://www.example.com`, when decoded, that is what is run.
+- **Shell scripts**. For example, if a user uploads and executes a script, the
+  commands run within the script are not captured, simply the output.
+- **Terminal controls**. Terminals support a wide variety of controls including
+  the ability for users to disable terminal echo. This is frequently used when
+  requesting credentials. Disabling terminal echo allows commands to be run
+  without being captured.
+
+Furthermore, due to their unstructured nature, session recordings are difficult to
+ingest and perform monitoring and alerting on.
+
+Teleport Enhanced Session Recording mitigates all three concerns by providing
+advanced security, greater logging capabilities, and better correlates a user
+with their activities.
+
+## Step 1/2. Configure a Teleport Node
```

Comment:
This whole section is mostly prerequisites and not related to enhanced recording. Is the intent that every guide involving SSH nodes should include steps for installing teleport, creating a join token, and joining the cluster? Or can we just document that process in one place and have guides assume you have a node running?

Reply:
I wanted to make this more like our Database Access guides (e.g., https://goteleport.com/docs/database-access/guides/mongodb-atlas/) where we include as much information as possible in a single guide so users don't need to break focus to follow the guide (https://goteleport.com/docs/contributing/documentation/style-guide/#how-to-guides).